client/my-sites/plugins/plugins-list/plugins-list-dataviews.tsx

```diff
@@ -0,0 +1,79 @@
+import { DataViews, filterSortAndPaginate } from '@wordpress/dataviews';
+import { useTranslate } from 'i18n-calypso';
+import React, { useEffect, useMemo, useState } from 'react';
+import { initialDataViewsState } from 'calypso/a8c-for-agencies/components/items-dashboard/constants';
+import { DataViewsState } from 'calypso/a8c-for-agencies/components/items-dashboard/items-dataviews/interfaces';
+import { useSelector } from 'calypso/state';
+import { Plugin } from 'calypso/state/plugins/installed/types';
+import { getPluginActionStatuses } from '../plugin-management-v2/utils/get-plugin-action-statuses';
+import { useActions } from './use-actions';
+import { useFields } from './use-fields';
+
+import './style.scss';
+
+interface Props {
+	currentPlugins: Array< Plugin >;
+	initialSearch?: string;
+	isLoading: boolean;
+	onSearch?: ( search: string ) => void;
+	bulkActionDialog: ( action: string, plugins: Array< Plugin > ) => void;
+}
+
+export default function PluginsListDataViews( {
+	currentPlugins,
+	initialSearch,
+	isLoading,
+	onSearch,
+	bulkActionDialog,
+}: Props ) {
+	const allStatuses = useSelector( ( state ) => {
+		return getPluginActionStatuses( state );
+	} );
```

Comment:
No need to wrap the selector in another selector:

```suggestion
	const allStatuses = useSelector( getPluginActionStatuses );
```

Reply:
TIL, [updated it here](https://github.com/Automattic/wp-calypso/pull/95561/commits/04e61486e434796d0da8518c874ec3949d4f3c9d). Thanks!